File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/01.First-Steps-In-Coding/02-exercise/08.basketballEquipment.js
function basketballEquipment(input) {

    const fee = Number(input[0]);

    const sneakersPrice = fee * 0.6;
    const outfitPrice = sneakersPrice * 0.8;
    const ballPrice = outfitPrice / 4;
    const accessoriesPrice = ballPrice / 5;
    const totalSum = sneakersPrice + outfitPrice + ballPrice + accessoriesPrice + fee;

    console.log(totalSum);

}
basketballEquipment(["550"]);

File: Python_Basics/01.first_steps_in_coding/02.exercise/08.basketball_equipment.py
fee = int(input())

# • Баскетболни кецове – цената им е 40% по-малка от таксата за една година
# • Баскетболен екип – цената му е 20% по-евтина от тази на кецовете
# • Баскетболна топка – цената ѝ е 1 / 4 от цената на баскетболния екип
# • Баскетболни аксесоари – цената им е 1 / 5 от цената на баскетболната топка 
snickers_price = fee * 0.6
outfit_price = snickers_price * 0.8
ball = outfit_price / 4
accessories_price = ball / 5

needed_sum = fee + snickers_price + outfit_price + ball + accessories_price

print(needed_sum)